core/vcpu.c

```diff
@@ -2041,6 +2041,10 @@ static void vcpu_enter_fpu_state(struct vcpu_t *vcpu)
     struct fx_layout *hfx = (struct fx_layout *)hax_page_va(hstate->hfxpage);
     struct fx_layout *gfx = (struct fx_layout *)hax_page_va(gstate->gfxpage);
 
+    vcpu->cr0_ts = !!(get_cr0() & CR0_TS);
+
```

Comment:
nit: Add a comment to explain why CLTS is necessary.

```suggestion

    // Before executing any FPU instruction (e.g. FXSAVE) in host kernel
    // context, make sure host CR0.TS = 0, so as to prevent a Device Not Found
    // (#NM) exception, which can result in a host kernel panic on NetBSD.
```